commec/utils/logger.py

```diff
@@ -44,7 +43,32 @@ def _find_message_start(self):
         return sample_formatted.find(sample.msg)
 
     def format(self, record):
+        """
+        Custom formattor for Commec purposes.
+        Accepts keywords in the extra dictionary:
+        one_line : skips text wrapping.
+        no_fmt : Skips the "WARNING | " prefixes.
+        cap, cap_up, cap_down : Neatly ties off formatted prefixes.
```

Comment:
```suggestion
        Custom formatter for Commec logging.  Accepts the following keywords in the extra dictionary:
        no_wrap : skips text wrapping.
        no_prefix: Skips the "%(asctime)s│ %(levelname)-8s│ % " prefixes.
        box, box_up, box_down: Use unicode box-drawing characters ( "─┘" or "─┐") to tie off formatted prefixes when switching to no-prefix lines.
```
(this makes it clearer to me what's going on, but if you feel like this names are less clear, feel free to reject)